nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const meals = require("./../data/meals.json");
+//Respond with the json for the meal with the corresponding id
+router.get("/:id", async(request, response) => {
+  try {
+      const { id } = request.params;
+      const mealId = meals.filter((meal) => meal.id === parseInt(id));
+      if (mealId < 1) {
```

Comment:
Operator '<' cannot be applied to types '{ id: number; title: string; maxNumberOfGuests: number; description: string; createdAt: string; price: number; }[]' and 'number'.ts(2365)